parsing/extend.ml

```diff
@@ -101,7 +101,7 @@ type ('self, 'a) symbol =
 | Aself : ('self, 'self) symbol
 | Anext : ('self, 'self) symbol
 | Aentry : 'a entry -> ('self, 'a) symbol
-| Aentryl : 'a entry * int -> ('self, 'a) symbol
+| Aentryl : 'a entry * string -> ('self, 'a) symbol
```

Comment:
What's the reason for this?

Reply:
OK, it just seems weird to see all the new `int_of_string`, also these are no guarded so we can get an anomaly. So I guess I am missing something.